vite.config.js

```diff
@@ -70,6 +71,11 @@ export default ({ mode }) => {
       mode !== 'lib' && uno({
         presets: [presetWarp()],
       }),
+      mode === 'development' && uno({
```

Comment:
Wont you need these classes when running in lib mode?

Reply:
are you referring to the "safeList: classes" that I've added for the development mode, and that it should also be in the lib-mode?

Reply:
> Wont you need these classes when running in lib mode?

We checked that when you run `pnpm dev`, the mode is 'development'. The goal with this additional uno setup was to test changes from the @warp-ds/css repo locally without too much effort (WarpElement superclass made it a bit tricky).
So if you have some classes that have changed in @warp-ds/css and you want to test them in @warp-ds/elements before making any releases, you could link that local repo here using `npm link`, add `@unocss-placeholder` to styles of the component you're testing, and this way you can see your new styles when running `pnpm dev`. We just need to remember to remove the `@unocss-placeholder` when we're done testing. I hope that makes sense 😊

Reply:
I have now updated the ReadMe-file so that it is clearer on the use of `@unocss-placeholder` and to inform that we need to remove it once we're done testing in this commit: [eb79307](https://github.com/warp-ds/elements/pull/107/commits/eb793076d71a7da601bf67cae4cb13dccccd0bdd) :)